02-peer-protocol.md

```diff
@@ -191,11 +188,11 @@ This message contains a transaction input.
 2. data:
     * [`channel_id`:`channel_id`]
     * [`u64`:`serial_id`]
-    * [`u16`:`prevtx_len`]
+    * [`bigsize`:`prevtx_len`]
```

Comment:
i dont know if `bigsize` has meaning in the spec? we could def define it; i  think tu64 might be more appropriate? But I don't think you can have more than one of those in a message.

Reply:
It does, it's defined in the fundamental types: https://github.com/lightning/bolts/blob/master/01-messaging.md#fundamental-types

It's not the same as a `tu64`, it's bitcoin's `bigsize` (also called `varint` in bitcoin), so you can have as many as you want in a message.

Reply:
fun random fact about '`varint`' -- bitcoin core's `varint` is NOT this, instead it's called `compactSize`. Every *other* implementation that I could think to check has it listed as `varint` though, so it seems reasonable to refer to it as such here also, thought the pedant in me is extremely non-plussed lol.